linkinator.config.json

```diff
@@ -3,7 +3,10 @@
   "skip": [
     "https://codecov.io/gh/googleapis/",
     "www.googleapis.com",
-    "img.shields.io"
+    "img.shields.io",
```

Comment:
I thought this file was templated, trying to figure out why OwlBot isn't swinging through and overwriting this on you 😄